rest/taskana-rest-spring-example/src/test/java/pro/taskana/doc/api/MonitorControllerRestDocumentation.java

```diff
@@ -77,31 +76,37 @@ public void setUp() {
                 fieldWithPath("sumRow.total").description("Total number of tasks"),
                 fieldWithPath("_links.self.href").ignored()
         };
-        
-        countByStateFieldDescriptors = new FieldDescriptor[] {
-                fieldWithPath("[]..state").description("The state the tasks are in"),
-                fieldWithPath("[]..counter").description("Number of tasks in the corresponding state")
-        };
     }
     
     @Test
     public void getTaskStatusReport() throws Exception {
         this.mockMvc.perform(RestDocumentationRequestBuilders
-                .get("http://127.0.0.1:" + port + "/v1/monitor/taskStatusReport")
+                .get("http://127.0.0.1:" + port + "/v1/monitor/tasks-status-report")
                 .header("Authorization", "Basic YWRtaW46YWRtaW4="))
         .andExpect(MockMvcResultMatchers.status().isOk())
         .andDo(MockMvcRestDocumentation.document("GetTaskStatusReportDocTest",
                 responseFields(taskStatusReportFieldDescriptors)));
     }
     
     @Test
-    public void getCountByState() throws Exception {
+    public void tasksWorkbasketReport() throws Exception {
         this.mockMvc.perform(RestDocumentationRequestBuilders
-                .get("http://127.0.0.1:" + port + "/v1/monitor/countByState?states=READY,CLAIMED,COMPLETED")
+                .get("http://127.0.0.1:" + port + "/v1/monitor/tasks-workbasket-report?daysInPast=4&states=READY,CLAIMED,COMPLETED")
                 .accept("application/hal+json")
-                .header("Authorization", "Basic dGVhbWxlYWRfMTp0ZWFtbGVhZF8x"))
+                .header("Authorization", "Basic YWRtaW46YWRtaW4="))
         .andExpect(MockMvcResultMatchers.status().isOk())
-        .andDo(MockMvcRestDocumentation.document("GetCountByStateDocTest",
-                responseFields(countByStateFieldDescriptors)));
+        .andDo(MockMvcRestDocumentation.document("GetTaskStatusReportDocTest",
```

Comment:
this call is not necessary, since you overwrite the snippet for the given identifier (which is already defined in getTaskStatusReport()) [documentation: https://docs.spring.io/spring-restdocs/docs/current/api/org/springframework/restdocs/mockmvc/MockMvcRestDocumentation.html#document-java.lang.String-org.springframework.restdocs.snippet.Snippet...-] 

Please remove

Reply:
I tried but it doesn't generate any diffent snippet for the others

Reply:
What do you mean by this? I see differen snippets